plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx

```diff
@@ -33,6 +33,18 @@ export const PaymentGateways = ( {
 	setupPlugin,
 	isFetching,
 }: PaymentGatewaysProps ) => {
+	const [ storeCountry, setStoreCountry ] = useState( 'US' );
```

Comment:
Nitpick: Perhaps we have a constant for this somewhere? Ignore if not.

Reply:
Unfortunately, we don’t have one at the moment. This will be addressed in a follow-up PR.